commands-v2/ramsete/robotcontainer.py

```diff
@@ -0,0 +1,140 @@
+from commands2 import RunCommand, RamseteCommand
+from commands2.button import JoystickButton, Button
+
+from wpilib import XboxController
+from wpilib.controller import RamseteController, PIDController
+
+from wpimath.controller import SimpleMotorFeedforwardMeters
+
+from wpilib.kinematics import ChassisSpeeds
+
+from wpilib.interfaces import GenericHID
+
+from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
+from wpimath.trajectory import TrajectoryConfig, TrajectoryGenerator, Trajectory
+
+from wpimath.geometry import Pose2d, Rotation2d, Translation2d
+
+from subsystems.drivetrain import Drivetrain
+
+import constants
+
+
+class RobotContainer:
+
+    """
+    This class hosts the bulk of the robot's functions. Little robot logic needs to be
+    handled here or in the robot periodic methods, as this is a command-based system.
+    The structure (commands, subsystems, and button mappings) should be done here.
+    """
+
+    def __init__(self):
+
+        # Create the driver's controller.
+        self.m_driverController = XboxController(constants.kDriverControllerID)
+
+        # Create an instance of the drivetrain subsystem.
+        self.m_robotDrive = Drivetrain(self.m_driverController)
+
+        # Configure and set the button bindings for the driver's controller.
+        self.configureButtons()
+
+        # Set the default command for the drive subsystem. It's default command will allow
+        # the robot to drive with the controller.
+
+        self.m_robotDrive.setDefaultCommand(
+            RunCommand(self.m_robotDrive.arcadeDrive, self.m_robotDrive)
+        )
+
+    # Return the autonomous command, the RAMSETE command.
+    def getAutonomousCommand(self):
+
+        # Create a voltage constraint to ensure we don't accelerate too fast.
+
+        autoVoltageConstraint = DifferentialDriveVoltageConstraint(
+            SimpleMotorFeedforwardMeters(
+                constants.ksVolts,
+                constants.kvVoltSecondsPerMeter,
+                constants.kaVoltSecondsSquaredPerMeter,
+            ),
+            constants.kDriveKinematics,
+            10,  # 10 volts max.
+        )
+
+        """Below will generate the trajectory using a set of programmed configurations"""
```

Comment:
You can just make this one a regular comment, I think. Docstrings only work at the beginning of a module/class/function.

Reply:
Ok, will do.